test/powershell/Host/Logging.Tests.ps1

```diff
@@ -43,6 +43,29 @@ enum LogKeyword
     ManagedPlugin = 0x100
 }
 
+# mac log command can emit json, so just use that
+# we need to deconstruct the eventmessage to get the event id
+# we also need to filter out the non-default messages
+function Get-MacOsSyslogItems {
```

Comment:
Given that this is to replace `Get-PSOsLog`, does it make sense to put this function in `PSSysLog.psm1`?

Reply:
@daxian-dbw I did think about that, but wanted to take more time to do a more complete rewrite of this module. I thought this smaller, localized change would be a faster path to stability, so I took this parallel approach.

Reply:
Sounds good to me 👍